pom.xml

```diff
@@ -246,6 +246,11 @@
         <artifactId>slf4j-log4j12</artifactId>
         <version>${slf4j.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Not necessarily, the other option is to add this in underfs/pom.xml
@apc999  Which one is Alluxio convention?